Content.Shared/StationRecords/GeneralStationRecord.cs

```diff
@@ -56,4 +56,10 @@ public sealed class GeneralStationRecord
     /// </summary>
     [ViewVariables]
     public int DisplayPriority;
+
+    /// <summary>
+    ///     Fingerprint of the person.
+    /// </summary>
+    [ViewVariables]
+    public string? Fingerprint = string.Empty;
```

Comment:
this should just be `Fingerprint;` since it will say it has a fingerprint of "" rather than N/A

Reply:
done

Reply:
oh wait im dumb i read incorrectly lol

Reply:
you made it required rather that default to null, now N/A can't be shown

Reply:
yea i see, its resolved now